Add initial Vagrant setup

ryanlerch opened the pull request:
Add an initial vagrant setup for datanommer.

This installs the 3 packages into a virtualenv.

No configuration setup yet, as that is all still fedmsg based

Resolves: #120

ryanlerch committed:
Add initial Vagrant setup

Add an initial vagrant setup for datanommer.

This installs the 3 packages into a virtualenv.

No configuration setup yet, as that is all still fedmsg based

Resolves: #120

Signed-off-by: Ryan Lerch <rlerch@redhat.com>

abompard commented on devel/ansible/roles/datanommer/tasks/main.yml:
```diff
@@ -0,0 +1,54 @@
+  become: yes
+  become_user: vagrant
+
+- name: Tell poetry not to create a virtualenv
```
I'm not sure that's necessary, poetry should not create an additional virtualenv when it's running from a virtualenv already

abompard replied:
Which means that we should source `/srv/venv/bin/activate` from the vagrant user's `.bashrc`.

ryanlerch replied:
Fair enough -- was just looking at fasjson's vagrant setup, and in there we alias /srv/venv/bin/potery to poetry: https://github.com/fedora-infra/fasjson/blob/dev/ansible/roles/fasjson/files/.bashrc

does that achieve the same thing? or would  you rather activating the venv with source in the bashrc?

abompard replied:
I think it's pretty much the same. But it feels like enabling the virtualenv is closer to the "nominal path" than making an alias and setting the poetry configuration. Maybe it's just me :-) Both would probably work.

ryanlerch commented:
Added two more commits to this one implemented the requested changes.

Will squash them and move out of WIP -- just keeping commits to make it easier to see what i have changed.
